Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -98,6 +98,7 @@ namespace Multiplayer
         "The base used for blending between network updates, 0.1 will be quite linear, 0.2 or 0.3 will "
         "slow down quicker and may be better suited to connections with highly variable latency");
     AZ_CVAR(bool, bg_multiplayerDebugDraw, false, nullptr, AZ::ConsoleFunctorFlags::Null, "Enables debug draw for the multiplayer gem");
+    AZ_CVAR(bool, sv_host_onstartup, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Whether to host as soon as the Multiplayer system becomes available.");
```

Comment:
I'd be down for naming this `sv_host` but I see the cl version also has a '_onstartup' prefix 😕

Reply:
Wouldn't we want dedicated server targets that have the multiplayer gem enabled to just always call host? Putting this particular command in a setreg file seems redundant. I could totally see the default map being in a cfg or setreg file though.

Reply:
That seems reasonable to make it true by default for dedicated servers.

Reply:
Fixed! Dedicated servers will automatically host unless told otherwise (sv_dedicated_host_onstartup=false)